rasa/graph_components/converters/nlu_message_converter.py

```diff
@@ -24,8 +24,7 @@ def create(
         """Creates component (see parent class for full docstring)."""
         return cls()
 
-    @classmethod
```

Comment:
Class methods don't work for graph component run methods 😁

Reply:
yeah, I've also discovered that 😁